config/package/tmpfiles.d/uwsgi.conf

```diff
@@ -0,0 +1 @@
+d /run/uwsgi/socorro 0755 socorro nginx -
```

Comment:
> 

When the RPM is built via scripts/install.sh this conf file will get dropped into /etc/tmpfiles.d/uwsgi.conf. Unfortunately, according to the packaging guidelines, this is an inappropriate behaviour - instead, said conf should be dropped into /usr/lib/tmpfiles.d/.
>
> Interestingly, this would be easier to deal with if we were using a real spec. In any case, we'll need > to modify the behaviour of scripts/install.sh if we want to adhere to the guidelines.
>
> Thoughts?

I think we're already fairly impure, just by virtue of using `fpm` and shipping a virtualenv (the "right" way would be to have RPM dependencies packaged properly for every one of our ~70 Python dependencies)

That said, we tried to do this right for systemd service files (which have a similar "RPM-provided files go to /usr/lib, human overrides go to /etc", so let's try to do it right here too.